crates/turborepo-lib/src/commands/info.rs

```diff
@@ -0,0 +1,93 @@
+use std::{env, io};
+
+use sysinfo::{System, SystemExt};
+use thiserror::Error;
+use turborepo_repository::{package_json::PackageJson, package_manager::PackageManager};
+
+use super::CommandBase;
+use crate::{DaemonConnector, DaemonConnectorError};
+
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("could not get path to turbo binary: {0}")]
+    NoCurrentExe(#[from] io::Error),
+}
+
+pub async fn run(base: CommandBase) -> Result<(), Error> {
```

Comment:
ITG (non blocking follow up): Separate the data from the display logic to make this more unit-test friendly and support additional output formats like JSON.

Something like:
```
struct TurboInfo {
    package_manager: String,
    daemon_status: String,
    ...
}
```

And then have `turbo_info.print()` to actually write the data to `stdout`